samples/cpp/common/utils/include/samples/common.hpp

```diff
@@ -1038,3 +1043,178 @@ static UNUSED void printPerformanceCounts(ov::InferRequest request,
     auto performanceMap = request.get_profiling_info();
     printPerformanceCounts(performanceMap, stream, deviceName, bshowHeader);
 }
+
+static UNUSED void printPerformanceCountsNoSort(std::vector<ov::ProfilingInfo> performanceData,
+                                          std::ostream& stream,
+                                          std::string deviceName,
+                                          bool bshowHeader = true) {
+    std::chrono::microseconds totalTime = std::chrono::microseconds::zero();
+    // Print performance counts
+    if (bshowHeader) {
+        stream << std::endl << "performance counts:" << std::endl << std::endl;
+    }
+    std::ios::fmtflags fmt(std::cout.flags());
+
+    for (const auto& it : performanceData) {
+        if (it.real_time.count() > 0) {
+            totalTime += it.real_time;
+        }
+    }
+
+    for (const auto& it : performanceData) {
+        std::string toPrint(it.node_name);
+        const int maxLayerName = 30;
+
+        if (it.node_name.length() >= maxLayerName) {
+            toPrint = it.node_name.substr(0, maxLayerName - 4);
+            toPrint += "...";
+        }
+
+        stream << std::setw(maxLayerName) << std::left << toPrint;
+        switch (it.status) {
+        case ov::ProfilingInfo::Status::EXECUTED:
+            stream << std::setw(15) << std::left << "EXECUTED";
+            break;
+        case ov::ProfilingInfo::Status::NOT_RUN:
+            stream << std::setw(15) << std::left << "NOT_RUN";
+            break;
+        case ov::ProfilingInfo::Status::OPTIMIZED_OUT:
+            stream << std::setw(15) << std::left << "OPTIMIZED_OUT";
+            break;
+        }
+        stream << std::setw(30) << std::left << "layerType: " + std::string(it.node_type) + " ";
+        stream << std::setw(20) << std::left << "realTime: " + std::to_string(it.real_time.count() / 1000.0) + " ms ";
+        stream << std::setw(15) << std::left << "cpu: " + std::to_string(it.cpu_time.count() / 1000.0) + " ms ";
```

Comment:
Can we set the format and reduce the number of digits after the comma? (especially for proportion)?
```
Res_layer1_1/add              EXECUTED       layerType: FullyConnected     realTime: 0.001000 ms cpu: 0.001000 ms proportion: 3.703704 % execType: jit_gemm_FP32
Res_layer0/add                EXECUTED       layerType: FullyConnected     realTime: 0.001000 ms cpu: 0.001000 ms proportion: 3.703704 % execType: jit_gemm_FP32
Res_layer1/add                EXECUTED       layerType: FullyConnected     realTime: 0.001000 ms cpu: 0.001000 ms proportion: 3.703704 % execType: jit_gemm_FP32
Res_layer0_1/add              EXECUTED       layerType: FullyConnected     realTime: 0.001000 ms cpu: 0.001000 ms proportion: 3.703704 % execType: jit_gemm_FP32
```

Also please align format between C++ and python

Reply:
on going it